default @bindir@ expands to ${exec_prefix}/bin and default @exec_prefix@
expands to ${prefix}

--- src/withsctp/withsctp.in
+++ src/withsctp/withsctp.in
@@ -1,5 +1,7 @@
 #!/bin/sh
 # -*- sh -*-
+prefix=@prefix@
+exec_prefix=@exec_prefix@
 LIBDIR=@libdir@/@PACKAGE@
 BINDIR=@bindir@
 export LD_PRELOAD=${LIBDIR}/libwithsctp.so.1.0.6
